examples/ristretto/src/ristretto.rs

```diff
@@ -266,33 +305,73 @@ pub fn encode(u: RistrettoPoint) -> RistrettoPointEncoded {
     RistrettoPointEncoded::new().update_start(&s.to_byte_seq_le())
 }
 
+/// Decodes the given point in accordance with the IETF standard.
+/// Note: There are many byte-strings resulting in incorrect decodings.
+/// These all checked for, in accordance with the IETF standard.
```

Comment:
This looks good, but in general it may be helpful to include direct links.
E.g. https://www.ietf.org/archive/id/draft-irtf-cfrg-ristretto255-00.html#name-decode
This could be done in a follow up PR, if you prefer that.

Reply:
```suggestion
/// These all checked for, in accordance with the IETF standard.
/// See https://www.ietf.org/archive/id/draft-irtf-cfrg-ristretto255-decaf448-03.html#name-decode
```